cmd/operator.go

```diff
@@ -71,6 +74,10 @@ func main() {
 			Usage: "Port on which the operator metrics are to be exposed",
 			Value: defaultMetricsPort,
 		},
+		cli.BoolFlag{
```

Comment:
instead of having a bool flag, can we take in the `essentials entitlement id`? This will solve 2 things
1. We do not need to ask the user to create the secret for metering
2. We know that the user has registered on the central SaaS portal before trying to install PX Essentials.
Also, taking the ID, will make sure they actually unlink any clusters that are running essentials for their user ID.

Reply:
These are the operator arguments and not the StorageCluster itself. In operator model users deploy two things - 
- Install the operator
- Create StorageCluster CR which has the portworx cluster configuration. Operator looks at StorageCluster and then creates a Portworx cluster accordingly

This boolean is meant to tell the operator that even if the user does not say that this has to be an essentials cluster, we create one because this is essentials operator.